.github/workflows/release.yml

```diff
@@ -4,23 +4,72 @@ on:
   pull_request:
     paths:
     - '.github/workflows/release.yml'
+    - 'scripts/create_spec_repo/*'
   schedule:
     # Run every day at 11pm (PST) - cron uses UTC times
     - cron:  '0 7 * * *'
 
 jobs:
+  buildup_SpecsTesting_repo:
+    # Don't run on private repo unless it is a PR.
+    if: github.repository != 'FirebasePrivate/firebase-ios-sdk' || github.event_name == 'pull_request'
+    env:
+      bot_token_secret: ${{ secrets.GHASecretsGPGPassphrase1 }}
+      local_repo: specstesting
+      local_sdk_repo_dir: /tmp/test/firebase-ios-sdk
+      podspec_repo_branch: master
+    runs-on: macOS-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Get token
+      run: scripts/decrypt_gha_secret.sh scripts/gha-encrypted/bot-access.txt.gpg \
+          bot-access.txt "$bot_token_secret"
+    - name: Update SpecStaging repo setup
+      run: scripts/release_testing_setup.sh
+    - name: Update SpecStaging repo
+      run: |
+        botaccess=`cat bot-access.txt`
+        cd scripts/create_spec_repo/
+        swift build
+        pod repo add --silent "${local_repo}" https://"$botaccess"@github.com/FirebasePrivate/SpecsTesting.git
+        BOT_TOKEN="${botaccess}"  .build/debug/SpecRepoBuilder --sdk_repo "${local_sdk_repo_dir}" --local_spec_repo_name "${local_repo}"
+    - name: Clean Artifacts
+      if: ${{ always() }}
+      run: |
+        pod repo remove "${local_repo}"
+        rm -rf bot-access.txt
+    - uses: actions/upload-artifact@v2
```

Comment:
How is the uploaded data used?

Reply:
The updated repo, with a "test" tag, will be transferred to testing jobs, which will use those repo as a source. Podspecs in the spec repo will have source leading to that repo and pods in quickstart could be tested.